.github/workflows/windows-qt6.yml

```diff
@@ -27,10 +27,10 @@ jobs:
         with:
           fetch-depth: 2
 
-      - name: 🐩 Install CMake and Ninja
-        uses: lukka/get-cmake@latest
-        with:
-          cmakeVersion: 3.29.6
+      - name: Check cmake version
+        shell: bash
+        run: |
+          which cmake && cmake --version
```

Comment:
```suggestion
```

Debug statement

Reply:
ok @m-kuhn, I misunderstand according to this comment (https://github.com/qgis/QGIS/pull/61330#discussion_r2028439580) you want to keep the echo of cmake version!

I will remove the statement.